apps/server/src/becca/becca_loader.ts

```diff
@@ -143,6 +148,8 @@ eventService.subscribeBeccaLoader([eventService.ENTITY_DELETED, eventService.ENT
         attributeDeleted(entityId);
     } else if (entityName === "etapi_tokens") {
         etapiTokenDeleted(entityId);
+    } else if (entityName === "note_embeddings") {
```

Comment:
Also added logic for the `note_embeddings` to work for the "Fill entity record changes" functionality